packages/broker/src/plugins/operator/MaintainOperatorValueService.ts

```diff
@@ -0,0 +1,81 @@
+import { Logger } from '@streamr/utils'
+import { MaintainOperatorValueHelper } from "./MaintainOperatorValueHelper"
+import { OperatorServiceConfig } from './OperatorPlugin'
+
+const logger = new Logger(module)
+
+const CHECK_VALUE_INTERVAL = 1000 * 60 * 60 * 24 // 1 day
+
+export class MaintainOperatorValueService {
+    private readonly config: OperatorServiceConfig
+    private checkValueInterval: NodeJS.Timeout | null = null
```

Comment:
It would maybe improve readability if most relevant fields are listed first? E.g. state fields before dependencies and those before configurations?

Reply:
Updated